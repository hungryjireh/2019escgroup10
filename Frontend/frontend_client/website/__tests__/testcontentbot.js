import React from "react";
import { render, cleanup, fireEvent, waitForElement, getByLabelText, getByText, getByTestId } from 'react-testing-library';
import ContentBot from "../components/Hero/Bot/ContentBot";
import {BotFace, BotText} from "../components/Hero/Bot/ContentBot";

afterEach(cleanup);

it("BotFace renders", () => {
    const { asFragment } = render(<BotFace />);
    expect(asFragment()).toMatchSnapshot();
});

it("BotText renders", () => {
    const { asFragment } = render(<BotText />);
    expect(asFragment()).toMatchSnapshot();
});