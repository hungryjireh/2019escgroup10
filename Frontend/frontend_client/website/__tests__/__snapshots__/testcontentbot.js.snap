// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BotFace renders 1`] = `
<DocumentFragment>
  <img
    class="sc-bwzfXH eAGVZw"
  />
</DocumentFragment>
`;

exports[`BotText renders 1`] = `
<DocumentFragment>
  <p
    class="sc-htpNat dkMODp"
  />
</DocumentFragment>
`;
